src/Utopia/Messaging/Adapters/SMS/Msg91.php

```diff
@@ -31,17 +33,28 @@ public function getMaxMessagesPerRequest(): int
         return 1000;
     }
 
+    public function setTemplate(string $templateId): self
+    {
+        $this->templateId = $templateId;
+
+        return $this;
+    }
+
     /**
      * {@inheritdoc}
      *
      * @throws \Exception
      */
     protected function process(SMS $message): string
     {
-        $to = \array_map(
-            fn ($to) => ['mobiles' => \ltrim($to, '+')],
```

Comment:
We probably still need this trim because in Appwrite, the phone number has the +

Reply:
Added